packages/users/src/api/index.ts

```diff
@@ -73,6 +75,52 @@ export interface GetThreadResponseObj {
   id: ThreadID
 }
 
+
+export interface UsageObj {
```

Comment:
Is the use of `Obj` in the object names a convention you guys have been using? Feels a bit cumbersome.

Reply:
well... it was not to conflict with the Usage type that comes back from the protos. However, and given the protos we really return Usage.AsObject obj result so went with it.